src/Main.hs

```diff
@@ -159,6 +163,7 @@ cliRun filePath debug compiled mode showStats hideQuotes strArgs = do
     Normalize -> do
       core <- doExtractCoreAt rxAt book 0
       return [(doLiftDups core)]
+  runtimeEnd <- getMonotonicTimeNSec
```

Comment:
since Extract and Collapse are lazy, there's no guarantee that they'll run to the end when you measure here. Especilly for collapse, printing is what causes values to be needed, so you can't measure it beforehand